package.json

```diff
@@ -88,5 +88,34 @@
   "dependencies": {
     "@types/lodash.clonedeep": "^4.5.6",
     "lodash.clonedeep": "^4.5.0"
+  },
+  "exports": {
```

Comment:
Exports require special treatment for mixing ESM and CommonJS (Our bundle package is UMD for example)

`
For more information about supporting both CommonJS and ES Modules in a single package please consult the dual CommonJS/ES module packages section.
`

Did we test to ensure that still works?

Reply:
tested on svelte + svelte ssr and updated "." value accordingly, please test on more frameworks